### PR TITLE
RB-325 add README

### PR DESCRIPTION
**Pull request by ziklodi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,13 +27,98 @@ bundle add sincerely
...
+## Configuration
+1. You need to set the notification model generated in the previous step by setting the `notification_model_name` option in the `config/sincerely.yml` file:
+
+```
+delivery_methods:
+  email:
+    class_name: Sincerely::DeliveryMethods::EmailAwsSes
+    options:
+  sms:
+```
```
Comment (willianveiga) [inline README.md:53]: Is this snippet here? Shouldn't it be part of the step 2?

File: README.md [hunk 2]
```diff
@@ -23,13 +27,98 @@ bundle add sincerely
...
+
+2. You need to set the delivery method for each delivery options (email/sms/push) by setting the `delivery_methods` option in the `config/sincerely.yml` file:
+
+```
+notification_model_name: Notification
+```
```
Comment (willianveiga) [inline README.md:59]: Is this snippet here? Shouldn't it be part of the step 1?

File: README.md [hunk 3]
```diff
@@ -23,13 +27,98 @@ bundle add sincerely
...
+
+## Usage
+1. For a detailed description of configuring AWS SES email sending see the following blog post:
```
Comment (willianveiga) [inline README.md:62]: I assume the blog post link is being added later, right?

Comment (willianveiga) [reply README.md:62]: Or is this blogpost an AWS reference?

File: README.md [hunk 4]
```diff
@@ -23,13 +27,98 @@ bundle add sincerely
...
+  delivery_options: {
+    template_data: {
+      name: 'John Doe'
+    },
+    subject: 'subject'
+  }
+)
+```
+* recipient: recipient of the notification
+* notification_type: email|sms|push
```
Comment (willianveiga) [inline README.md:98]: Probably we want to add an * saying something like `right now the gem supports only AWS SES emails`.

File: README.md [hunk 5]
```diff
@@ -23,13 +27,98 @@ bundle add sincerely
...
+* rejected: notification is rejected by the delivery system
+* delivered: notification is delivered by the delivery system
+* bounced: notification is bounced (ie when an email cannot be delivered to the recipient)
+* opened: notification is opened
+* clicked: notification is clicked
+* complained: notification is complained (ie when the recipient of your email marks your email as spam)
+
+## Callbacks
+
+For a detailed description of AWS SES email callbacks see the following blog post
```
Comment (willianveiga) [inline README.md:120]: Blog post missing here too.

**Comment by willianveiga** [comment]
Do we want to describe how to create custom template classes and custom delivery method classes?

File: README.md (hunk 3 above)
Comment (ziklodi) [reply README.md:62]: > I assume the blog post link is being added later, right?

correct

**Comment by ziklodi** [comment]
> Do we want to describe how to create custom template classes and custom delivery method classes?

aAt this point we should not, I think.

**Review by willianveiga — APPROVED** [review]

**Pull request merged**